src/main/java/astvisitors/Typechecker.java

```diff
@@ -4,23 +4,20 @@
 import ast.arduino.*;
 import ast.expr.*;
 import ast.expr.aexpr.AExpr;
+import ast.expr.cast.DoubleCastNode;
+import ast.expr.cast.IntegerCastNode;
 import ast.funcallstmt.CustomFuncCallStmtNode;
 import ast.funcallstmt.PrintNode;
-import ast.funcallstmt.cast.DoubleCastNode;
-import ast.funcallstmt.cast.IntegerCastNode;
 import ast.type.DoubleLiteral;
 import ast.type.IdNode;
 import ast.type.IntegerLiteral;
 import ast.type.StringLiteral;
 import exceptions.ErrorHandler;
 
-import java.util.Arrays;
-
-public class Typechecker extends AstVisitor {
-    private final String keywords[] = {"PRINT", "RETURN", "DEFAULT", "SWITCH"};
+public class TypeChecker extends AstVisitor {
```

Comment:
File named Typechecker and class name is TypeChecker

Reply:
Fixed TypeChecker and TypeCheckerTest